platforms.rb

```diff
@@ -129,8 +129,15 @@
   remap "el"
   version_remap do |opts|
     # A combo of Fedora 19/20 was used to create RHEL 7 so when we're on Fedora 20 use the RHEL 7 packages
```

Comment:
Should this comment move down above line 136?

Reply:
Moved